content/writing/blog-posts.md

```diff
@@ -23,6 +23,9 @@ When writing a blog, use your outline as a guide and expand on all points as nee
 
 We are looking forward to this and are excited to read what you write!!!
 
+## Images
+You’ll need a feature image: the image will be used on the front page, and as the "hero" image on your blog post. The size needs to be around 2:1 (twice larger than the height) to fit our WordPress template. The size of the images in the article doesn’t matter, but always use the highest possible quality: they will be automatically optimized using a lossless encoding). Lastly, always add a text for the ALT description of all of your images. Be as descriptive as you can: ask yourself how you would like someone to describe them to you if you were blind.

```

Comment:
`The size of the images in the article doesn’t matter, but always use the highest possible quality: they will be automatically optimized using a lossless encoding ) ` There is an extra bracket here

Reply:
I think it would use `hero` for words we want to highlight or make emphasis